### ISSUE TITLE
Unexpected error happened when mouse down happened outside of canvas

### ISSUE THREAD

**Issue by Journey** [simaQ/g2-brush#7]
# Repro steps:
1. Press the mouse down outside of the canvas area,
2. Hold the mouse and move into the canvas area,
3. Release the mouse

# Result
There is an error:
`Uncaught TypeError: Cannot read property 'x' of null
at e._onCanvasMouseUp (g2-brush.js:485)....`

# Reason
It is caused by `_onCanvasMouseUp` need `startPoint`, which should be assigned in `_onCanvasMouseDown`. But it is not triggerred in step 1, because mousedown happened outside of canvas area.

# Suggestions
* check `startPoint` in `_onCanvasMouseUp`, if null, then return.
* If `mousedown` happened outside of `canvas`, do the things like ` _onCanvasMouseDown` while  enter the canvas area.

**Comment by web3max** [comment]
you can use `inPlot: false`

**Comment by fix777** [comment]
> you can use `inPlot: false`

Yeah, it works for me. Thanks. But what's the meaning of the API `inPlot`, it's confusing in the [API Docs](https://github.com/antvis/g2-brush/blob/master/docs/brush.md).
